src/MongoDB/Driver/Manager.cpp

```diff
@@ -53,6 +53,7 @@ const StaticString
 	s_MongoDBDriverManager_readpreferencetags("readpreferencetags"),
 	s_MongoDBDriverManager_readPreference("readPreference"),
 	s_MongoDBDriverManager_readPreferenceTags("readPreferenceTags"),
+	s_MongoDBDriverManager_maxStalenessMS("maxStalenessMS"),
```

Comment:
Why do you define these strings with multiple cases? URI option parsing should be case-insensitive, per [the spec](https://github.com/mongodb/specifications/blob/master/source/connection-string/connection-string-spec.rst#key-value-pair):

> Keys should be normalised and character case should be ignored.